README.md

```diff
@@ -16,7 +16,15 @@ translated to [Gateway API](https://gateway-api.sigs.k8s.io/) directly.
 
 Note: Ingress2gateway is not intended to copy annotations from Ingress to Gateway API.
 
-## Build from Source
+## Installation
+
+If you have a Go development environment locally, you can install ingress2gateway with `go install github.com/kubernetes-sigs/ingress2gateway@v0.1.0`
```

Comment:
Does it mean that we need to update the version when new releases?

Reply:
it does